test/unicode_parameter_test.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: iso-8859-1 -*-
+#
+# Copyright 2012-2015 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from helpers import unittest
+
+import luigi
+
+
+class UnicodeParameterTest(unittest.TestCase):
```

Comment:
Actually you shouldn't need to create a whole new file for this I think. Can't you just delete this file and move this test to `parameter_test.py`?